moto/sagemaker/models.py

```diff
@@ -3396,6 +3404,14 @@ def create_model_package(
         sample_payload_url: Any,
         additional_inference_specifications: Any,
     ) -> str:
+        if not bool(model_package_group_name) ^ bool(model_package_name):
```

Comment:
Can we simplify this to `if model_package_group_name and model_package_name`? That would be more readable

Reply:
Yeah! Sure, good call!

Reply:
Hmm, I just noticed that we **need** the XOR because `model_package_name` is now `Optional`.

I feel that this XOR operation is better than `if model_package_name and model_package_group_name or not model_package_name and not model_package_group_name`. 

What do you think?

Reply:
Hmm, I think it is better to treat this as another if-statement as the message is different if none of them are passed. I will fix it asap.